Package.swift

```diff
@@ -334,7 +334,7 @@ var targets: [Target] = [
 ]
 
 #if TUIST
-    import struct ProjectDescription.PackageSettings
+    @preconcurrency import struct ProjectDescription.PackageSettings
```

Comment:
This should also not be necessary as `PackageSettings` conforms to `Sendable` 🤔

Reply:
Thanks! https://github.com/tuist/tuist/pull/6738/commits/316375ebba41bfcf73e92d7d37322b2e286ce938